Dockerfile.node-cache

```diff
@@ -12,11 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ARG_FROM
```

Comment:
Shouldn't this be added back if we want to use this base image and use iptables-legacy?

Reply:
It become `FROM ARG_FROM_IPT` to use the iptables base image